frontend/packages/vkt/src/components/publicEnrollment/steps/PersonDetails.tsx

```diff
@@ -20,21 +21,40 @@ export const PersonDetails = () => {
     <div className="rows gapped">
       <H3>{t('title')}</H3>
       <div className="grid-columns gapped">
-        <CustomTextField
-          label={t('lastName')}
-          value={person.lastName}
-          disabled
-        />
-        <CustomTextField
-          label={t('firstName')}
-          value={person.firstName}
-          disabled
-        />
-        <CustomTextField
-          label={t('identityNumber')}
-          value={person.identityNumber}
-          disabled
-        />
+        {person.lastName && person.firstName ? (
+          <>
+            <CustomTextField
+              label={t('lastName')}
+              value={person.lastName}
+              disabled
+            />
+            <CustomTextField
+              label={t('firstName')}
+              value={person.firstName}
+              disabled
+            />
+          </>
+        ) : (
+          <CustomTextField
+            label={t('name')}
+            value={person.firstName}
+            disabled
+          />
+        )}
+        {person.identityNumber && (
```

Comment:
Suomalaisen e-tunnisteen kautta ei taida tulla tuota syntymäaikaa koskaan hetun ohessa? Mietin että jos sattuis tulemaan näytettäis 4 kenttää rinnakkain (etunimet, sukunimi, hetu, syntymäaika).

Reply:
Ei ainakaan noilla testitunnuksilla näytä tulevan. Mutta jos tulis neljä kenttää niin ei pitäis olla leiskan puolesta ongelma. 

<img width="1138" alt="Näyttökuva 2023-5-4 kello 9 01 58" src="https://user-images.githubusercontent.com/435495/236123602-12ed79e4-ed7c-473d-b456-ab763a68802e.png">

Reply:
Joo, voihan tuohonkin skenaarioon tosiaan varautua